homeassistant/loader.py

```diff
@@ -65,6 +65,7 @@
 # This list can be extended by calling async_register_preload_platform
 #
 BASE_PRELOAD_PLATFORMS = [
+    "backup",
     "config",
     "config_flow",
     "diagnostics",
```

Comment:
Side note: logbook below is misplaced.

Reply:
[7e3abdf](https://github.com/home-assistant/core/pull/133905/commits/7e3abdfbd85e8abfa2f6020191fe20ec00fd5a22)